test/benchmarks/micro/spinner.inputs

```diff
@@ -0,0 +1,18 @@
+1
```

Comment:
What is the format of this file?

Reply:
Can you leave a brief README in the benchmarking directory to explain the format and how it works?